manim/renderer/opengl_renderer_window.py

```diff
@@ -14,7 +16,10 @@ class Window(PygletWindow):
     cursor = True
 
     def __init__(self, renderer, size=config.window_size, **kwargs):
-        monitors = get_monitors()
+        if platform.system() == "Darwin":
```

Comment:
This seems fine to me if we patch this, but I know @naveen521kk was saying it should be fixed upstream. This PR seems to fix it and is approved so might be better waiting for screeninfo to merge and not add extra logic to manim ? https://github.com/rr-/screeninfo/pull/61

Reply:
yes! Indeed @rr- was quick to answer and fix it, we will be releasing soon, then I'll revert this commit and just update the dependency

Reply:
Great! I think waiting for the fix from upstream is the correct approach here; I'll convert this PR into a draft in the meantime. Thank you for your contribution!

Reply:
I just released docstring-parser 0.8 which should fix this issue. Thank you @betafcc for great contributions :)